docs/src/examples/components/Tree/Types/TreeExample.shorthand.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import { Tree } from '@stardust-ui/react'
+const treeData = [
```

Comment:
An empty line before `const treeData`, please
```suggestion

const treeData = [
```